client/src/main/webjar/program-form-dialog/program-form-dialog.module.js

```diff
@@ -8,6 +8,7 @@
             "healthBam.templates",
             "healthBam.program",
             "healthBam.organization",
+            "healthBam.programArea",
```

Comment:
Adding the new module to the app here, but not using it the form yet (that's next).